src/styles/global.css

```
@@ -1,97 +1,270 @@
+  --space-xl: 2rem;
+  --space-2xl: 3rem;
+  --space-3xl: 4rem;
+  --space-section: 6rem;
+
+  /* Layout */
+  --max-width: 1200px;
+  --content-width: 65ch;
+  --header-height: 4rem;
+
+  /* Shadows */
+  --shadow-sm: 0 1px 2px 0 rgb(0 0 0 / 0.05);
```

Stylelint (17.2.0)

[error] Expected "0.05" to be "5%" (alpha-value-notation)

(alpha-value-notation)